src/CheckboxItem/CheckboxItem.js

```diff
@@ -227,6 +227,13 @@ module.exports = kind(
 	/**
 	* @private
 	*/
+	detectTextDirectionality: function () {
+		this.applyStyle('direction', null);
```

Comment:
Curious if we need to execute `applyStyle`, or if we could just make this method a no-op.